vendor/github.com/crifff/private/main.go

```diff
@@ -0,0 +1,5 @@
+package private
+
+func Hello() string {
```

Comment:
__[gometalinter]__
* [golint] exported function Hello should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->